root/components/common-macros.tt

```diff
@@ -828,15 +828,17 @@ END -%]
 [%- END -%]
 
 [%~ MACRO medium_description(medium) BLOCK;
-    args = {
+    format_position = l('{medium_format} {position}', {
         medium_format => medium_format_name(medium),
         position => medium.position,
-    };
+    });
     IF medium.name != '';
-        args.medium_name = isolate_text(medium.name);
-        l('{medium_format} {position}: {medium_name}', args);
+        l('{medium_format_position} {medium_name}', {
+            medium_format_position => '<span>' _ add_colon(format_position) _ '</span>',
```

Comment:
I like how the HTML was moved out of the string. 👍

I guess this'll allow translators to change the order of `{medium_format_position}` and `{medium_name}`, and maybe add additional words for context. Does that make sense if these are supposed to be in separate cells?

There seems to be a contradiction between having these cells visually separated from each other, but allowing translators to edit the string as if it's all in one cell. So maybe we shouldn't have this "combined" string anymore?